netci.groovy

```diff
@@ -178,7 +178,7 @@ commitPullList.each { isPr ->
 
   def triggerPhraseOnly = false
   def triggerPhraseExtra = "microbuild"
-  Utilities.setMachineAffinity(myJob, 'Windows_NT', 'latest-or-auto-dev15')
+  Utilities.setMachineAffinity(myJob, 'Windows_NT', 'latest-or-auto-dev15-rc')
```

Comment:
I thought these machines were Dev14 + Dev15RC.  If it's Dev15 only then yeah can't do this for a few of these jobs.